examples/gradle-v2/kotlin-multiplatform-example/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-    kotlin("multiplatform") version "1.9.25"
+    kotlin("multiplatform") version "2.1.0"
```

Comment:
WDYT about creating separate PR to update all other examples to use latest Kotlin version?

Reply:
Yeah, I think that's a good idea.

Reply:
created PR: https://github.com/Kotlin/dokka/pull/3949